python/Glacier2/callback/README.md

```diff
@@ -13,54 +13,30 @@ flowchart LR
     g --connection1--> s[Server:4061<br>hosts WakeUpService] --connection2--> g
 ```
 
-We recommend running each program in a separate Python virtual environment.
-If you are new to Python virtual environments, see [Python Virtual Environments].
-
-## Ice prerequisites
+## Prerequisites
 
+- [uv] Python package and project manager.
```

Comment:
This should be:
```
- Install uv. See [Installing uv].
```

Where Installing uv = https://docs.astral.sh/uv/getting-started/installation/